data.sql

```diff
@@ -32,4 +32,34 @@ INSERT INTO  owners(full_name,age) VALUES('Dean Whinchester',14);
 INSERT INTO owners(full_name,age) VALUES('Jodie Whittaker',38);
 -- INSERTION TO SPECIES TABLE
 INSERT INTO species(name) values ('Pokemon');
-INSERT INTO species(name) VALUES ('Digimon');
\ No newline at end of file
+INSERT INTO species(name) VALUES ('Digimon');
+
+-- insertion to vets table
+INSERT INTO vets(name,age,date_of_graduation) VALUES('William Tatcher',45,'2000-04-23');
+INSERT INTO vets(name,age,date_of_graduation) values('Maisy Smith',26,'2019-01-17');
+INSERT INTO vets(name,age,date_of_graduation) values('Stephanie Mendez',64,'1981-05-04');
+insert INTO vets(name,age,date_of_graduation) values('Jack Harkness',38,'2008-06-08');
+
+-- insertion to specialites table
+
+
+-- insertion to visits table
+insert into visits(vets_id,animal_id,date_of_visit) values(1,1,'2020-05-24');
+insert into visits(vets_id,animal_id,date_of_visit) values(3,1,'2020-07-22');
+insert into visits(vets_id,animal_id,date_of_visit) values(4,2,'2021-02-02');
+insert into visits(vets_id,animal_id,date_of_visit) values(2,3,'2020-01-05');
+insert into visits(vets_id,animal_id,date_of_visit) values(2,3,'2020-05-14');
```

Comment:
- [ ] Please add provided data in the `specialization` table

Reply:
ohhh ok ok sorry